sw/device/tests/consecutive_irqs_test.c

```diff
@@ -209,6 +198,11 @@ static bool plic_configure_irqs(dif_plic_t *plic) {
 }
 
 static bool execute_test(dif_uart_t *uart) {
+  // Initialize the global variables.
+  uart_rx_overflow_handled = false;
+  uart_tx_empty_handled = false;
+  uart_handled_more_than_once = false;
```

Comment:
You shouldn't need to do this; the OT CRT should be doing this for you.

Reply:
Filed #2160 to track this.